modules/ingest-geoip/src/main/java/org/elasticsearch/ingest/geoip/direct/GetDatabaseConfigurationAction.java

```diff
@@ -77,27 +69,36 @@ public boolean equals(Object obj) {
             Request other = (Request) obj;
             return Arrays.equals(databaseIds, other.databaseIds);
         }
+
     }
 
-    public static class Response extends ActionResponse implements ToXContentObject {
+    public static class Response extends BaseNodesResponse<NodeResponse> implements ToXContentObject {
 
         private final List<DatabaseConfigurationMetadata> databases;
 
-        public Response(List<DatabaseConfigurationMetadata> databases) {
-            this.databases = List.copyOf(databases); // defensive copy
+        public Response(
+            List<DatabaseConfigurationMetadata> databases,
+            ClusterName clusterName,
+            List<NodeResponse> nodes,
+            List<FailedNodeException> failures
+        ) {
+            super(clusterName, nodes, failures);
+            this.databases = databases;
```

Comment:
You lost the defensive copy here -- does that matter?

Reply:
That was an accident. Putting the copy back.